controller/events/dispatcher_metrics.go

```diff
@@ -134,8 +134,9 @@ func (self *Dispatcher) unregisterMetricsEventHandler(val interface{}) {
 
 func (self *Dispatcher) newMetricEvent(msg *metrics_pb.MetricsMessage, metricType string, name string, id string) *event.MetricsEvent {
 	result := &event.MetricsEvent{
-		MetricType:    metricType,
 		Namespace:     event.MetricsEventsNs,
+		EventSrcId:    self.network.GetAppId(),
```

Comment:
And now "app id" == event source id == index id?

Reply:
app id == event source id == the id of the controller. They are distinct from the raft/datastore index.

Reply:
OH. I see. I was drawing on my event sourcing background which has an "event source id" which is the "index" in those systems. But this is literally the "id of the source" not the "id of the event sourced event"